docs/release_notes/v-1.6.md

```diff
@@ -0,0 +1,109 @@
+
+### Schema configuration updates in the connection Timeline
+
+When someone makes a change to a connection's schema, Airbyte now logs an event in the connection's Timeline page. This data can help you understand why sync behaviors or record volumes suddenly changed. [**Learn more >**](../cloud/managing-airbyte-cloud/review-connection-timeline)
+
+![A connection Timeline showing a different count of records after a schema change](assets/1-6-connection-timeline.png)
+
+### Copy JSON from connectors
+
+All connectors now have a **Copy JSON** button in the Airbyte UI. This button generates a JSON object matching the configuration of that connector. It's ready for use in Airbyte's API and Terraform provider. [**Learn more >**](../terraform-documentation#weakly-typed-json-configurations)
+
+```json title="Airbyte source connector example"
+{
+  "name": "Airbyte",
+  "workspaceId": "c0e5b294-2c71-475b-ae9c-6d70b36ff4f4",
+  "definitionId": "284f6466-3004-4d83-a9b2-e4b36cbbbd41",
+  "configuration": {
+    "client_id": "******", // Replace this with your real client ID.
+    "start_date": "2024-01-01T00:00:00Z",
+    "client_secret": "******" // Replace this with your real client secret.
+  }
+}
+```
+
+### Connector Builder improvements
+
+Airbyte's Connector Builder has new features to target more advanced and complex API sources: asynchronous endpoints, non-REST APIs, new response formats, and more.
+
+#### Asynchronous streams
+
+The Connector Builder UI now supports asynchronous streams, allowing you to handle scenarios where data extraction happens over multiple steps. Some APIs that handle large datasets or perform resource-intensive operations do not support synchronous requests. By setting up a stream as asynchronous, you can separate creating a request, polling for a response, and downloading the response into distinct steps. [**Learn more >**](../connector-development/connector-builder-ui/async-streams)
+
+![Connector Builder UI showing an async stream with separate tabs for Creation, Polling, Download, and Schema](assets/1-6-asynchronous-streams.png)
+
+#### GraphQL request body
+
+You can write request bodies as free form GraphQL queries with built-in syntax validation.
+
+![Connector Builder UI showing a GraphQL query in the request body](assets/1-6-graphql.png)
+
+#### Handle CSV, gzip, and ZIP response formats
+
+Airbyte has two new ways to handle API responses in the Connector Builder.
+
+- **CSV decoding**: Make requests to sources that respond in CSV format and set expected delimiter and encoding options.
+
+- **Nested decoding**. Make requests to sources that respond with compressed `gzip` or `zip` files, then set a nested decoding option to handle the unzipped contents within.
+
+[**Learn More >**](../connector-development/connector-builder-ui/record-processing/#response-decoding)
+
+![A stream that returns a zip file with a CSV inside](assets/1-6-decoders.png)
+
+### Create custom connectors from YAML or Docker images
```

Comment:
Maybe specify that those are new API endpoints in the title.

Are they available in TF? if so we should probably mention it there.

Reply:
@ian-at-airbyte agree -- lets add 'in the API'. In the description, let's add it's available in terraform too, and just link to the terraform provider (we don't need to provide exact instructions)